.travis.yml

```diff
@@ -41,51 +44,48 @@ before_install:
         ;;
     esac
 
-    pip install pylint==1.8.1
-    pip install futures==3.1.1
-    pip install grpcio==1.6.3
-
 script:
   - |
-    bazel \
-      --output_base="${HOME}/.bazel-output-base" \
-      --batch \
-      --host_jvm_args=-Xmx500m \
-      --host_jvm_args=-Xms500m \
-      build \
-      tensorboard/... \
-      --worker_verbose \
-      --verbose_failures \
-      --spawn_strategy=sandboxed \
-      --genrule_strategy=sandboxed \
-      --local_resources=400,2,1.0 \
-      --worker_max_instances=2 \
-      --distinct_host_configuration=false
+    # Configure Bazel
+    {
+      # Storing build artifacts in this directory helps Travis cache them. This
+      # will sometimes cut latency in half, when we're lucky.
+      echo "startup --output_base=${HOME}/.bazel-output-base"
 
-  - |
-    bazel \
-      --output_base="${HOME}/.bazel-output-base" \
-      --batch \
-      --host_jvm_args=-Xmx500m \
-      --host_jvm_args=-Xms500m \
-      test \
-      tensorboard/... \
-      --verbose_failures \
```

Comment:
Do we not want `--verbose_failures` and `--test_verbose_timeout_warnings` any more?

Reply:
Yes indeed we do. I'm also seeing if I can shave some latency off the Bazel install.